inc/namespace.php

```diff
@@ -31,3 +33,42 @@ function on_plugins_loaded() {
 		require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
 	}
 }
+
+function qm_file_path_map( $map ) : array {
+	// Chassis and Local Server
+	if ( file_exists( '/etc/chassis-constants' ) ) {
+		$json_string = file_get_contents( '/etc/chassis-constants' );
+		$data = json_decode( $json_string, true );
+		if ( ! empty( $data['synced_folders']['/chassis'] ) ) {
```

Comment:
`$data` could be `null` if the JSON file is garbled, need to check that here too.

Also, I think this needs to handle other paths. Can we instead loop over `synced_folders` and ensure all are handled? Should be able to basically merge it directly in I think

Reply:
if `$data` is null then we'd have `! empty( null )` which should resolve to `false`. I set the value explicitly to `null` to check